server/src/main/java/com/google/musicanalysis/site/RequestBuilder.java

```diff
@@ -0,0 +1,78 @@
+package com.google.musicanalysis.site;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/*
+    Handles gathering data from the Musixmatch API.
+    This is structured the way it is becuase I will be adding more
+    to each function.
+*/
+public class RequestBuilder {
+
+  // Url building
+  private String baseUrl = "https://api.musixmatch.com/ws/1.1/";
```

Comment:
These are constants that will _never_ change. Make them static and final, and name them in `SCREAMING_SNAKE_CASE`.

```suggestion
  private static final String BASE_URL = "https://api.musixmatch.com/ws/1.1/";
```